tests/integration/metrics_test.go

```diff
@@ -211,3 +215,34 @@ func TestMetricsHealth(t *testing.T) {
 		t.Fatalf("expected '0' from etcd_server_health_failures, got %q", hv)
 	}
 }
+
+func TestMetricsRangeDurationSeconds(t *testing.T) {
+	integration.BeforeTest(t)
+	clus := integration.NewCluster(t, &integration.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	client := clus.RandClient()
+
+	keys := []string{
+		"my-namespace/foobar", "my-namespace/foobar1", "namespace/foobar1"}
+	for _, key := range keys {
+		_, err := client.Put(context.Background(), key, "data")
+		require.NoError(t, err)
+	}
+
+	// Range over all keys.
```

Comment:
I'm unsure if it's just me, but I find this comment confusing. I'm not sure if it refers to ranging over the keys (in the for loop, and it got misplaced). Or if it refers to the metrics range.

Reply:
Good point. The comment seems ambiguous now. I will remove it shortly.